definitions/ext-environment_sensor/summary_metrics.yml

```
@@ -20,3 +20,9 @@ uptime:
       from: Metric
       where: "provider = 'kentik-watchdog'"
       eventId: entity.guid
+    # Sunbird PowerIQ devices
+    kentik/watchdog:
```

You can not repeat same kentik/watchdog: under queris

---

@albertoolivan is correct, if you change it as below it should work (just checked the validation on my local):
```suggestion
    kentik/power-iq:
```

---

nice catch; thanks!